CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to
 - ✨(oidc) add simple introspection backend #832
 - 🧑‍💻(tasks) run management commands #814
 
+### Fixed
+
+- 🐛(oauth2) force JWT signed for /userinfo #804
```

Comment:
Wrong change information here

Reply:
Indeed, copy-paste fumble… will fix in a later PR, thanks